### ISSUE TITLE
Update to Lyo 4.0

### ISSUE THREAD

**Issue by berezovskyi** [eclipse/lyo.testsuite#19]

**Comment by jamsden** [comment]
This is the OSLC4J OSLC v2.0 test suite.

**Comment by berezovskyi** [comment]
Yes, I know. Lyo 4.0 does not support OSLC 3 and even if it were, we are not dropping OSLC 2 support? Still need to update to supported versions.

**Comment by berezovskyi** [comment]
Here is what we'd like to do to "update" to Lyo 4.0 (after digging deeper into the codebase):

- stop using Jena and Apache HttpClient directly where possible and switch to a Lyo Client.
